genie-web/src/main/java/com/netflix/genie/web/controllers/JobRestController.java

```diff
@@ -781,7 +781,9 @@ private void copyRequestHeaders(final HttpServletRequest request, final ClientHt
     private void copyResponseHeaders(final HttpServletResponse response, final ClientHttpResponse forwardResponse) {
         final HttpHeaders headers = forwardResponse.getHeaders();
         for (final Map.Entry<String, String> header : headers.toSingleValueMap().entrySet()) {
-            response.setHeader(header.getKey(), header.getValue());
+            if (!TRANSFER_ENCODING_HEADER.equalsIgnoreCase(header.getKey())) {
```

Comment:
I would add a comment here why you're doing this so that no one takes it out by accident going forward